tests/test_sklearn_normalizer_converter.py

```diff
@@ -61,6 +63,46 @@ def test_float64_normalizer_converter(self):
                 model.transform(data), torch_model.transform(data_tensor), rtol=1e-06, atol=1e-06,
             )
 
+    # ONNX backend
+    @unittest.skipIf(not (onnx_runtime_installed()), reason="ONNXML test requires ONNX and  ORT")
```

Comment:
Was this onnx test moved from somewhere else?

Also,  I know that this is different from our [onnxml normalizer test](https://github.com/microsoft/hummingbird/blob/master/tests/test_onnxml_normalizer_converter.py#L27), but it is hella confusing.

Actually, at this point, even I am confused, and i wrote that.  Can you please add comments clarifying?

Reply:
No this is because we were missing a test for this operator for the onnx backend (without onnx-ml).

Reply:
To add a little bit on this. The onnx tests (the one starting with the `onnx_` prefix) basically tests the onnx frontend (i.e., the onnx converters). But onnx can also be a backend, so this test makes sure that the path sklearn->pytorch->onnx works. At the end I hope we can get to a point where, for each operator, for each frontent, we a test for each different backend.

Reply:
Ok, this should at least say "ONNX test" not `ONNXML test`

Reply:
Broader point....this distinction is clear to us, because we have been staring at "onnx" vs "onnx-ml" for a few years now. (for better or worse..... :) ) But ... the average person walking into this project will be very confused, especially since onnx-ml is severely under-documented!  Is there a way that we can explain this in a way that makes the project less intimidating to newcomers?   (not necessarily right here, but maybe in the documentation somewhere?)

Reply:
Yes it should definitely say onnx test :)

I agree, we should explain this somewhere. Were you thinking to put this strictly in the documentation on the README or on a different document? Perhaps we can start to push some of this on the website.